### ISSUE TITLE
Translate to french persists

### ISSUE THREAD

**Issue by netrules** [luizdepra/hugo-coder#518]
# Translate to french issue

## Previous history
- Issue https://github.com/luizdepra/hugo-coder/issues/296 by me
- Pull https://github.com/luizdepra/hugo-coder/pull/318

## Current issue
After fixing the issue and committing the pull, it still persists, as described in the following image:
![image](https://user-images.githubusercontent.com/48886164/105910256-75a95000-6007-11eb-9642-09e0cd13819d.png)

The issue here is that the "*Google Chrome*" translator widget is picking up my logo, name and headline first which it detects as being french (no clue why, my name is in no way french, more like spanish... **wattah google**?)

Anyway, here's the solution I came up with:
![image](https://user-images.githubusercontent.com/48886164/105911051-81494680-6008-11eb-8c31-b266734c294c.png)

I have tried a bunch of other things. Meta language references, notranslate, and so on. Apparently it's a very dodgy (& sketchy) feature that only works as expected in common scenarios.

## How To Implement Fix
![image](https://user-images.githubusercontent.com/48886164/105911557-0fbdc800-6009-11eb-87bc-a82e04e54433.png)
And add to locales:
i.e spanish:
```toml
[translator_fix]
other = "Esto está en español."
```
english:
```toml
[translator_fix]
other = "This is in english."
```

## In Conclusion
Well. Just posting for reference in case anybody still has the same issue. Lmk if I should PR.
